### ISSUE TITLE
Question of fine turning

### ISSUE THREAD

**Issue by ioky31** [showlab/Show-o#64]
Thank you for your excellent work. I am planning to fine-tune show-o for a specific downstream task, I used medical image text dataset for stage3 fine-tuning, but I found that the generated result is not satisfactory, and the mode collapse, how can I solve this problem?

**Comment by Sierkinhane** [comment]
Can you show some examples and what's the dataset size you use to finetune?

**Comment by ioky31** [comment]
![Image](https://github.com/user-attachments/assets/3dd94854-fdea-4f98-baef-beaace487354)
Approximately 90k cases of data

**Comment by ioky31** [comment]
![Image](https://github.com/user-attachments/assets/22069c24-6084-4b56-95cc-265c65bbf840)
Approximately 90k cases of data

**Comment by Sierkinhane** [comment]
The cfg is disabled when online inference during training. You'd try offline inference and enable the cfg.

Besides, the image on the first row is predicted in one step with a high mask ratio (0.87), which significantly degrades the generation quality.

**Comment by IKEJAY-code** [comment]
> ![Image](https://github.com/user-attachments/assets/22069c24-6084-4b56-95cc-265c65bbf840) Approximately 90k cases of data大约 90k 个数据案例

Hello brother, I also encountered the same problem. The fine-tuning results in the field of medical images are not satisfactory. May I ask if you have obtained a satisfactory result now?🤖
